tests/auxil/bot_method_checks.py

```diff
@@ -62,6 +72,33 @@ def check_shortcut_signature(
     Returns:
         :obj:`bool`: Whether or not the signature matches.
     """
+
+    def resolve_class(name: str) -> Optional[type]:
+        """Attempts to resolve a PTB class from a ForwardRef.
+
+        E.g. resolves <class 'telegram._files.sticker.StickerSet'> from "StickerSet".
+
+        Returns a class on success, :obj:`None` if nothing could be resolved.
+        """
+
+        for option in (
+            name,
+            f"telegram.{name}",
+            f"telegram.ext.{name}",
+            f"telegram.ext.filters.{name}",
+        ):
+            try:
+                return eval(option)
+            # NameError will be raised if trying to eval just name and it doesn't work, e.g.
+            # "Name 'ApplicationBuilder' is not defined".
+            # AttributeError will be raised if trying to e.g. eval f"telegram.{name}" when the
+            # class denoted by `name` actually belongs to `telegram.ext`:
+            # "module 'telegram' has no attribute 'ApplicationBuilder'".
+            # If neither option works, this is not a PTB class.
+            except (NameError, AttributeError):
+                continue
```

Comment:
```suggestion
            attr = getattr(module, name, None)
            if attr is not None:
                return None
```

I'd like to not rely on `eval` if not strictly necessary :)

Reply:
hmm... why are we not doing this in the admonitions?

Reply:
I don't understand the `return None` bit. We want to return a class.

Reply:
> then maybe return module.name?

yes ofc. my bad 🤦